lib/Twig/Node/Expression/Binary/StrictEqual.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+class Twig_Node_Expression_Binary_StrictEqual extends Twig_Node_Expression_Binary
+{
+    public function operator(Twig_Compiler $compiler)
+    {
+        return $compiler->raw('===');
```

Comment:
I'm confused, this is causing an exception being thrown with a message informing that `same as` should be used right? (https://github.com/twigphp/Twig/issues/2358#issuecomment-271995283)